ppr-ui/tests/unit/Debtors.spec.ts

```diff
@@ -144,3 +145,37 @@ describe('Debtor amendment tests', () => {
   })
 
 })
+
+
+describe('Debtor validation tests', () => {
+  let wrapper: Wrapper<any>
+
+  beforeEach(async () => {
+    await store.dispatch('setAddSecuredPartiesAndDebtors', {
+      debtors: mockedDebtorsDeleted
+    })
+    await store.dispatch('setRegistrationFlowType', RegistrationFlowType.AMENDMENT)
+    wrapper = createComponent()
+  })
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('displays the correct rows', () => {
+    const rowCount = wrapper.vm.$el.querySelectorAll('.v-data-table .debtor-row').length
+    // one greyed out row
+    expect(rowCount).toEqual(1)
+    const item1 = wrapper.vm.$el.querySelectorAll('.v-data-table .debtor-row')[0]
+    expect(item1.querySelectorAll('td')[0].textContent).toContain('removed')
+  })
+
+  
+  it('displays the error', async () => {
+    wrapper.vm.$props.setShowInvalid = true
+    expect(wrapper.vm.getDebtorValidity()).toBe(false)
+    wrapper.vm.$data.showErrorDebtors = true
+    await Vue.nextTick()
+    expect(wrapper.findAll('.invalid-message').length).toBe(1)
+  })
+
```

Comment:
Can you add one test where it starts valid and then updates to invalid afterwards (i.e. starts with one valid debtor that is then removed afterwards)

Reply:
done